examples/fixedin_ramp64.rs

```diff
@@ -0,0 +1,198 @@
+extern crate rubato;
+use rubato::{InterpolationParameters, InterpolationType, Resampler, SincFixedIn, WindowFunction};
+use std::convert::TryInto;
+use std::env;
+use std::fs::File;
+use std::io::prelude::{Read, Seek, Write};
+use std::io::Cursor;
+use std::time::Instant;
+
+extern crate env_logger;
+extern crate log;
+use env_logger::Builder;
+use log::LevelFilter;
+
+///! A resampler app that reads a raw file of little-endian 64 bit floats, and writes the output in the same format.
```

Comment:
It would be easier to use this example if it didn't require such a specific file format which isn't widely used. I suggest to use [Symphonia](https://lib.rs/crates/symphonia) then output as a WAV with [Hound](https://lib.rs/crates/hound) or [wav](https://lib.rs/crates/wav).

Reply:
The examples use this format to make it easy to read and write the data with numpy. I use the examples and a python script for validating the resampling results. 
If I want to listen to the output, I usually import the data to Audacity. It's just a few clicks extra to load that instead of a wav.

Reply:
What are you using to convert audio files to this format?

Reply:
Also Audacity. It can export as header-less raw.
https://manual.audacityteam.org/man/other_uncompressed_files_export_options.html